### PR TITLE
autoyast summary

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/lib/y2network/presenters/interface_summary.rb [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+      # @return [String]
+      attr_reader :name
+
+      # Constructor
+      #
+      # @param name [String] name of device to describe
+      # @param configs [Y2Network::ConnectionConfigsCollection]
+      # @param interfaces [Y2Network::InterfacesCollection]
+      def initialize(name, configs, interfaces)
```
[the remark this answers is not in the document]
Comment (jreidinger) [reply src/lib/y2network/presenters/interface_summary.rb]: yes makes sense and maybe replace it by one config that is passed.

File: src/lib/y2network/connection_config/base.rb [hunk 2]
```diff
@@ -126,6 +126,24 @@ def all_ips
...
+      def find_master(configs)
+        configs.find do |config|
+          # TODO: what about VLAN?
+          if config.type.bonding?
+            config.slaves.include?(name)
+          elsif config.type.bridge?
+            config.ports.include?(name)
+          end
+
+          false
```
[the remark this answers is not in the document]
Comment (jreidinger) [reply src/lib/y2network/connection_config/base.rb]: no, it is not needed.

File: src/lib/y2network/presenters/interface_summary.rb [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+          status = status_info(config)
+
+          bullets << _("Device Name: %s") % config.name
+          bullets << status
+          bullets << config.startmode.long_description
+          bullets += aliases_info(config)
+
+          if config.type.bonding?
+            label = _("Bonding Slaves")
```
Comment (imobachgs) [inline src/lib/y2network/presenters/interface_summary.rb]: What about a note for translators?

File: test/y2network/presenters/interface_summary_test.rb [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  let(:name) { "vlan1" }
+
+  let(:config) do
+    Y2Network::Config.new(
+      interfaces: interfaces, connections: connections, source: :testing
+    )
+  end
+  let(:interfaces) do
+    Y2Network::InterfacesCollection.new([
+                                          double(Y2Network::Interface, hardware: nil, name: "vlan1"),
```
Comment (imobachgs) [inline test/y2network/presenters/interface_summary_test.rb]: np: This indentation, although correct, looks weird.

File: test/y2network/presenters/interfaces_summary_test.rb [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  # - eth1 unconfingured
+  # - vlan1 on top of eth0
+  let(:config) do
+    Y2Network::Config.new(
+      interfaces: interfaces, connections: connections, source: :testing
+    )
+  end
+  let(:interfaces) do
+    Y2Network::InterfacesCollection.new([
+                                          double(Y2Network::Interface, hardware: nil, name: "vlan1"),
```
Comment (imobachgs) [inline test/y2network/presenters/interfaces_summary_test.rb]: np: Indentation looks weird.

**Pull request merged**